src/main/java/seedu/address/model/attendance/Week.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.model.attendance;
+
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.util.Objects;
+
+import seedu.address.model.lesson.NumberOfOccurrences;
+
+/**
+ * Represents a week number.
+ * Guarantees: immutable; is valid as declared in {@link #isValidWeek(int)}
+ */
+public class Week {
```

Comment:
This class seems unused, but perhaps we can convert it to store an `Index` while performing validation on the `Index` value.

This way, we can ensure that there will be no ambiguity when parsing, transferring or retrieving values based on weeks.